src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,41 @@
+/* #region menu */
+
+.menu {
+  height: 100vh;
+  box-sizing: border-box;
+  background-color: #f9f9f9;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 22px;
+
+  &__top {
+    @include menu-top-margin-bottom;
```

Comment:
![image](https://github.com/user-attachments/assets/e46ecde8-e42a-4b89-be92-d1b46aa21d9b)

I don't see any styles that makes it possible